db/mb_import.py

```diff
@@ -0,0 +1,33 @@
+import db
```

Comment:
let's rename this file to musicbrainz_importer and move it to `acousticbrainz-server/musicbrainz_importer/musicbrainz_importer.py`

Reply:
Added a new musicbrainz_importer folder.